### PR TITLE
QWIN-36177: Implement a chat room silence option

### PR DESCRIPTION
**Pull request by hakermi**
Implement a chat room silence option

### CODE REVIEW

File: application/src/main/webapp/vue-app/components/ExoChatContactList.vue [hunk 1]
```diff
@@ -94,8 +94,25 @@
...
+          </div>
+          <div>
+            <v-icon
+              color="red darken-2"
+              dense
+              class="ml-2 mb-1"
+              v-if="isRoomNotificationSilence(contact.room)">
+              mdi-volume-off
+            </v-icon>
+          </div>
```
Comment (AzmiTouil) [inline application/src/main/webapp/vue-app/components/ExoChatContactList.vue]: IMO, it should be moved to ExoChatContact.vue ans passing unreadTotal as prop
WDYT?

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: application/src/main/webapp/vue-app/components/ExoChatContactList.vue (hunk 1 above)
Comment (hakermi) [reply application/src/main/webapp/vue-app/components/ExoChatContactList.vue]: Ok

Comment (hakermi) [reply application/src/main/webapp/vue-app/components/ExoChatContactList.vue]: Done

**Review by AzmiTouil — APPROVED** [review]
LGTM

**Pull request merged**